src/SvgStream.h

```diff
@@ -50,12 +50,14 @@ class SvgStreamFile : public SvgStream {
   void write(const std::string& data) { stream_ << data; }
 
   void flush() {
-    stream_ << "</svg>\n";
-    stream_.seekp(-7, std::ios_base::cur);
+    stream_ << "</svg>";
+    stream_.seekp(-6, std::ios_base::cur);
     stream_.flush();
   }
 
   void finish() {
+    stream_ << "</svg>\n";
+    stream_.flush();
```

Comment:
Can you add a couple of comment in this file so we don't later recreate the problems?